lib/class-experimental-wp-widget-blocks-manager.php

```diff
@@ -245,6 +257,9 @@ public static function serialize_blocks( $blocks ) {
 	 * @return string String representing the block.
 	 */
 	public static function serialize_block( $block ) {
+		if( ! isset( $block[ 'blockName' ] ) ) {
+			return '';
```

Comment:
Usually in PHP one returns `false` when there's an error. It should not impact `serialize_blocks` because PHP automatically converts `false` to `''` when casting to a string.

```
php > var_dump( 'hello' . false . 'jorge' );
string(10) "hellojorge"
```